app/views/content/train-to-be-a-teacher/teacher-training-interview.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Prepare for your teacher training interview
+subcategory: Your teacher training application
+description: |-
+  Prepare for a teacher training interview. Learn more about what the interview involves, what you might be asked, and how to get help to prepare.
+related_content:
+    Improve your subject knowledge : "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    What teacher training is like : "/train-to-be-a-teacher/initial-teacher-training"
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+date: "2023-06-09"
+keywords:
+  - adviser
+  - advisor
+  - ITT
+  - Initial Teacher Training
+  - QTS
+  - Qualified Teacher Status
+  - Teacher Training Adviser
+  - application
+  - applying for teacher training
+  - process
+  - PGCE
+  - interviews
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-interviews
+navigation: 20.30
+navigation_title: Preparing for interviews
+navigation_description: Find out how to prepare for a teacher training interview, with tips on what's involved, common questions, and to how to get help.
+---
+
+Congratulations on being invited for an interview! This is an important stage in successfully getting a place on a teacher training course.
+
+These tips can help you prepare for a successful interview. 
+
+It could be an interview for a course that leads to qualified teacher status (QTS), a postgraduate certificate in education (PGCE), or both. 
+
+The exact interview process will vary depending on the teacher training provider.
+
+## Confirm your interview date
+
+The provider will email you with an interview date. If you can't make it then you may be able to ask for an alternative.
+
+Once the date is agreed you should receive a confirmation email from the provider. Check this carefully to make sure you know exactly what to expect. 
+
+Most providers ask for some type of presentation. Others may require maths or English tests to be completed in advance.
+
+## Prepare what you need to know
+
+It's a good idea to check the provider's website to understand more about them and what appeals to you. You can refer to this information at your interview.
+
+Consider some possible safeguarding scenarios and how you might address them. Make sure you've read the [statutory guidance for schools and colleges on safeguarding children](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2).
+
+Also, take some time to find out about the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) and the [national curriculum](https://www.gov.uk/government/collections/national-curriculum).
```

Comment:
One last question @philspray - should this sentence come above the safeguarding one (without the `Also,`!)?